src/wordpress/bin/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -f /var/www/html/wp-config.php ]; then
 	mv wp-cli.phar /usr/local/bin/wp;
 	cd /var/www/html;
 	wp core download --allow-root;
-	mv /var/www/wp-config.php /var/www/html/
+	mv /var/lib/wp/wp-config.php /var/www/html/
```

Comment:
so here using this directory makes it clearer the intent of this file, is that it?

Reply:
yes, and it is also a Linux convention to place application-specific data in `/var/lib`.

I've left `/var/www/html` not knowing how WordPress works. I assume WordPress picks up everything located in `/var/www/html` and executes it.